iothub/device/src/ModuleClient.ConventionBasedOperations.cs

```diff
@@ -37,5 +36,31 @@ public Task SendTelemetryAsync(TelemetryMessage telemetryMessage, CancellationTo
             Func<CommandRequest, object, Task<CommandResponse>> callback, object userContext,
             CancellationToken cancellationToken = default)
             => InternalClient.SubscribeToCommandsAsync(callback, userContext, cancellationToken);
+
+        /// <summary>
+        /// Retrieve the device properties.
```

Comment:
Same comment as for the device client method, but doubly so because this should at least be "module" instead of "device" if we don't want to say "Retrieve the client properties"

Reply:
I updated both to say "client", reduces chance of a similar copy-paste error in the future